### PR TITLE
Select respective option after redirection

### PR DESCRIPTION
**Pull request by jiangyue12392**
Now after submitting reviews from maps, `messages-loader.js` automatically set the option in the cinema dropdown list.

### CODE REVIEW

File: src/main/webapp/js/messages-loader.js [hunk 1]
```diff
@@ -84,6 +84,7 @@ function createCinemaOptions() {
         cinemaSelect.appendChild(option);
```
Comment (potay) [inline src/main/webapp/js/messages-loader.js:84]: Just wondering what's the benefit of using async/await vs putting the `cinemaSelect.value = cinemaKey;` line here in this callback instead?

Comment (jiangyue12392) [reply src/main/webapp/js/messages-loader.js:84]: I think you are right. I didn't think of it when I was implemented. Changed to put in callback for better consistency.

Comment (Reine0017) [reply src/main/webapp/js/messages-loader.js:84]: @jiangyue12392 think this was from a previous commit but line 76 has a missing ';' at the end.

Comment (jiangyue12392) [reply src/main/webapp/js/messages-loader.js:84]: I think we somehow missed it in previous reviews. I will correct it together here then

**Review by Meg1456 — APPROVED** [review]

**Pull request merged**